### DISCUSSION TITLE
Retry workflow example

### DISCUSSION THREAD

**Discussion by DanielDubeSp** [elsa-workflows/elsa-core#3988] [Q&A]
Hi all. I'm looking for some retry example with no luck.  My last goal is that a failed part of a workflow (I'm chaining workflows) can be auto-retried, maybe in a loop.. not sure about this. I can't use 'Retry' endpoint because I don't want to restart the entire workflow, only some chained workflow
Thanks in advance

**Comment by sfmskywalker** [comment]
Would you be looking for a retry activity, or a retry option on specific activities, or something else?

**Reply by DanielDubeSp** [reply]
Hi! Any example will help me a lot. Retry an entire workflow (calling Retry from UI or calling Retry endpoint) or retry an activity, for example. Currently what we do when our workflow fails in a step, we enqueue the failed workflow again, and it works fine. But maybe there is a more elegant way to do it. 
For example if WF1 calls WF2 and WF2 calls WF3, then when WF2 fails we queue again only WF2 and WF2 calls WF3.  
Thanks in advance for your help and congratulations for the product
